### ISSUE TITLE
`RETURN *` errors if only a path variable is in scope

### ISSUE THREAD

**Issue by DominicWuest** [RedisGraph/RedisGraph#3104]
When running the following query:
```cypher
MERGE x = () RETURN *
```

I encountered the following error:

```
RETURN * is not allowed when there are not variables in scope
```

However, replacing `RETURN *` with `RETURN x` lets the query run successfully, proving that `x` is a variable in scope.
Because of this, I believe that the query should run successfully.

I encountered this issue when testing queries on the **RedisGraph master branch** in a Docker container running **alpine v.3**.

### Steps to reproduce
Run the following query and observe an error is thrown:
```cypher
MERGE x = () RETURN *
```

### Expected behavior
The query should run successfully

### Actual behavior
The query fails with the error message `RETURN * is not allowed when there are no variable in scope`.

**Comment by LiorKogan** [comment]
Related: #2341

**Comment by raz-mon** [comment]
Hi @DominicWuest,
Thanks for reporting the issue.
It is solved in #2926.
